apps/main/src/layout/default/Header.tsx

```diff
@@ -69,7 +70,7 @@ export const Header = ({ sections }: HeaderProps) => {
             ],
             rLocalePathname,
             routerPathname,
-            routerNetwork,
+            rNetwork,
           ),
         [hasRouter, network, networks, rChainId, rLocalePathname, routerCached, routerNetwork, routerPathname],
```

Comment:
It's a bit confusing how there's now both `rNetwork` from the url and `routerNetwork` also from the url. Can this be harmonized / simplified?

Reply:
This actually was not working with routerNetwork

It is something to dig into